client/src/components/Navbar/Navbar.scss

```diff
@@ -37,7 +37,9 @@ li {
   justify-content: flex-start;
 }
 
-a {
+a,
```

Comment:
If you're trying to solve https://github.com/UofT-Frosh-Orientation/orientation-website/issues/110 you will need to do inline styles in the <Link> component itself